protocol/messenger_chats.go

```diff
@@ -67,7 +68,11 @@ func (m *Messenger) CreatePublicChat(request *requests.CreatePublicChat) (*Messe
 
 	// We set the synced to, synced from to the default time
 	if !willSync {
-		timestamp := uint32(m.getTimesource().GetCurrentTime()/1000) - defaultSyncInterval
+		defaultSyncPeriod, err := m.settings.GetDefaultSyncPeriod()
```

Comment:
I'd suggest extracting this piece of code to function like `InitChatSyncFields()`or something like that because we already use it 4 times.